native/src/main/java/tech/sourced/LexerDemo.java

```diff
@@ -0,0 +1,38 @@
+package tech.sourced;
+
+import com.ansorgit.plugins.bash.lang.BashVersion;
+import com.ansorgit.plugins.bash.lang.lexer.BashLexer;
+import com.intellij.psi.tree.IElementType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LexerDemo {
```

Comment:
It is no longer needed for the bblfsh driver, it only made sense when this was a demo of BashSupport. Removed.